src/types/dto/ApplicationForm.ts

```diff
@@ -13,7 +13,27 @@ export interface Question {
   required: boolean;
 }
 
-export interface ApplicationForm {
+export interface Team {
+  createdAt: string;
+  createdBy: string;
+  name: string;
+  teamId: number;
+  updatedAt: string;
+  updatedBy: string;
+}
+
+export interface ApplicationFormResponse {
+  applicationFormId: number;
+  createdAt: string;
+  createdBy: string;
+  name: string;
+  questions: Question[];
+  team: Team;
+  updatedAt: string;
+  updatedBy: string;
+}
+
+export interface CreateApplicationFormRequest {
```

Comment:
```suggestion
export interface ApplicationFormCreateRequest {
```

Create 는 동사라 뒤로 빼주는게 어떨까요?

Reply:
https://github.com/mash-up-kr/mash-up-admin-fe/pull/58/commits/a374875416bfff24c617d6f779c565fdcca862e6 수정헀습니다!